### ISSUE TITLE
Screen off only on laptop with external monitors

### ISSUE THREAD

**Issue by insomnes** [hyprwm/hypridle#30]
Using listener like this weirdly shutdowns only laptop display, external monitor keeps on:
```
# Turn display off on 450 seconds of inactivity
listener {
    timeout = 450
    on-timeout = hyprctl dispatch dpms off
    on-resume = hyprctl dispatch dpms on
}
```

**Comment by vaxerski** [comment]
that's a hyprland/wlr bug and is tracked somewhere there, use the search featuer

**Comment by insomnes** [comment]
In the wlroots repo you mean? 
I've searched hyprland, hyperlock repos also and can not find any related info.

**Comment by vaxerski** [comment]
https://github.com/hyprwm/Hyprland/issues?page=1&q=is%3Aissue+dpms

somewhere there, though again, it's not a hypridle or hyprlock issue, either hl or wlroots, more likely the latter. Especially if you have nvidia.

**Comment by insomnes** [comment]
I've tried searching and the only somewhat related issue is:
https://github.com/hyprwm/Hyprland/issues/1475

But it currently works fine from direct terminal invoke and disables all monitors.

The problem only arises in combination with hypridle.

**Comment by vaxerski** [comment]
> The problem only arises in combination with hypridle.

well mention that from the start then

**Comment by insomnes** [comment]
Yeah, I've done not the best work on describing my problem, sorry for consuming your time.
